modules/apps/journal/journal-content-web/src/main/java/com/liferay/journal/content/web/internal/display/context/JournalContentDisplayContext.java

```diff
@@ -358,6 +360,66 @@ public String getDDMTemplateKey() throws PortalException {
 			return _ddmTemplateKey;
 		}
 
+		if (FeatureFlagManagerUtil.isEnabled(
+				_themeDisplay.getCompanyId(), "LPD-27566")) {
+
+			String ddmTemplateExternalReferenceCode =
+				_journalContentPortletInstanceConfiguration.
+					ddmTemplateExternalReferenceCode();
+
+			String ddmTemplateKey = ParamUtil.getString(
```

Comment:
why not?
```
String ddmTemplateKey = ParamUtil.getString(
      _portletRequest, "ddmTemplateKey",
      _journalContentPortletInstanceConfiguration.ddmTemplateKey()); 
```

I can not be the old conf?

Reply:
No, because in this case we are using the template key when we are configuring the portlet because with the template selector we are getting the templateKey and not the templateExternalReferenceCode, and, for that reason we are getting the ddmTemplateKey from the request and not getting the templateKey from the configuration because we are using the templateExternalReferenceCode, so, that is the reason because we are not using this piece of code.

Reply:
A question: if you are coming from an old configuration to a new, Why the old values are not valid here?

Reply:
You have to work with the new values here, and for the other case, we have to prepare a upgrade process when we delete the FF in order to change the configuration from the old values to the new ones and in that moment we won't have old values in the configuration.